### PR TITLE
Add Math and Random helpers

### PR DESCRIPTION
**Pull request by rappie**
- Add more types to assertion wrapper functions
- Add Math and Random helpers

### CODE REVIEW

File: src/MathHelper.sol [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+abstract contract MathHelper {
+    function min(uint256 a, uint256 b) internal pure returns (uint256) {
+        return a < b ? a : b;
+    }
```
Comment (ljz3) [inline src/MathHelper.sol:7]: Should also add these:
```suggestion
    function min(uint256 a, uint256 b) internal pure returns (uint256) {
        return a < b ? a : b;
    }
    
      // Forked from https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/utils/math/SignedMath.sol
    function max(int256 a, int256 b) internal pure returns (int256) {
        return a > b ? a : b;
    }

    // Forked with modifications from https://ethereum.stackexchange.com/a/84391
    function abs(int128 n) internal pure returns (int128) {
        return n >= 0 ? n : -n;
    }
```

File: src/RandomHelper.sol [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+abstract contract RandomHelper {
+    /// @notice Shuffle an array using Fisher-Yates algorithm
+    /// @dev Based on https://gist.github.com/scammi/602387a22e04c77beb73c0ebc0f0bc18
+    function shuffleArray(uint256[] memory shuffle, uint256 entropy) internal {
```
Comment (ljz3) [inline src/RandomHelper.sol:7]: Need to return the array, since it's just being shuffled in memory

**Review by ljz3 — CHANGES_REQUESTED** [review]

File: src/RandomHelper.sol (hunk 2 above)
Comment (rappie) [reply src/RandomHelper.sol:7]: No, this actually works. I'm using it in one codebase :slightly_smiling_face:

Comment (ljz3) [reply src/RandomHelper.sol:7]: Alright sounds good 🙂

**Pull request merged**